src/os/bluestore/BlueStore.cc

```diff
@@ -9168,6 +9243,15 @@ int BlueStore::_fsck(BlueStore::FSCKDepth depth, bool repair)
   if (r < 0) {
     return r;
   }
+
+  if (repair) {
```

Comment:
One cannot repair bdev label in fsck.
To run fsck both bluefs and bluestore must be properly initialized.
Should bdev label be broken initialization will fail.

Repair of bdev label must be an external action.